### DISCUSSION TITLE
plugin插件加载chart  怎么指定用echarts渲染

### DISCUSSION THREAD

**Discussion by chauncey-zhang** [dream-num/Luckysheet#349] [General]
plugin插件加载chart  怎么指定用echarts渲染，现在默认是用chartmix.umd

**Comment by dream-num** [comment]
chartmix.umd.js是 [chartMix](https://github.com/mengshukeji/chartMix) 插件打包生成的核心代码，其中的渲染逻辑即用的echarts。
换句话说，不用指定，现在用的就是echarts。
